src/test/java/team/asd/dao/TestChannelPartnerDao.java

```diff
@@ -0,0 +1,25 @@
+package team.asd.dao;
+
+import team.asd.entity.ChannelPartner;
+
+public class TestChannelPartnerDao implements ChannelPartnerDao{
```

Comment:
@olena-syrotenko this class is not formatted

Reply:
done